cmd/modelcmd/controller.go

```diff
@@ -425,92 +425,80 @@ func (c *OptionalControllerCommand) Init(args []string) (err error) {
 // MaybePrompt checks if the command was give a --client or --controller options.
 // If not, it will prompt user to clarify whether the operation is to take place on
 // a client copy, a controller copy or both.
+// When neither client nor controller is specified on the command,
+// several scenarios need to be catered for when prompting:
+// 1. there is a current controller to prompt user with;
+// 2. there is no current controller but there are other controllers, so inform users how to use -c;
+// 3. there is no current controller and there are no registered controllers, so inform users to bootstrap or register.
+// When there is no current controller, the prompt becomes a Y/N client question instead of a multi-choice
```

Comment:
The other option for this last scenario is when the command is read only, ie list, show, just do it on the client and include in the output what was done. Adds/removes/updates warrant the prompt. Just a thought, could be done as an improvement next

Reply:
As an improvement next... But then again, feels like a potentially nice improvement for all the commands... When there is only one choice, why not just do it and inform the user? All our operations are reversable if user makes a mistake.